### PR TITLE
SOW-24: show latest status change in PlanViewer

### PR DESCRIPTION
**Pull request by vferraro-scottlogic**

### CODE REVIEW

File: src/context/FormContext.tsx [hunk 1]
```diff
@@ -153,37 +151,36 @@ export const FormProvider = (props: { children: ReactNode }) => {
...
+      setLoadedTimetableEvents(events);
+
+      const currentStatusChangeEvent = events?.find(
         ({ developmentPlanEvent, endDate }) =>
           isStatusChangeEventKey(developmentPlanEvent) && !endDate
       );
 
-      if (hasActiveStatusChange) {
+      if (currentStatusChangeEvent) {
         setStatusHasChanged(true);
```
Comment (osouthwell-scottlogic) [inline src/context/FormContext.tsx:162]: This is called setStatusHasChanged but its more like status is not "progressing normal", right? Just making sure I understand

Comment (Ro4052) [reply src/context/FormContext.tsx:162]: I'm not sure if I've understood your question but `statusHasChanged` directly maps to the answer to this question:
image

Comment (osouthwell-scottlogic) [reply src/context/FormContext.tsx:162]: Okay so if you load a paused timetable then a no on this page would keep it paused or un-pause it?

Comment (vferraro-scottlogic) [reply src/context/FormContext.tsx:162]: then "yes" will create a new status change event which then gets added in updatedTimetableEvents at the end on ExportPage

**Review by osouthwell-scottlogic — APPROVED** [review]

**Pull request merged**